main.py

```diff
@@ -1,10 +1,72 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_or_update_race(race_data: dict) -> Race:
+    race_description = race_data.get("description")
+    if race_description is None:
+        return (Race.objects.get_or_create(name=race_data["name"]))[0]
+    return (Race.objects.get_or_create(
+        name=race_data["name"],
+        description=race_description)
+    )[0]
+
+
+def create_or_update_guild(guild_data: dict) -> Guild | None:
+    if not guild_data:
+        return None
+    guild_description = guild_data.get("description")
+    return Guild.objects.get_or_create(
+        name=guild_data["name"],
+        description=guild_description
+    )[0]
+
+
+def create_or_update_skill(skill_data: dict, race: Race) -> None:
```

Comment:
`create_or_update_skill`? What exactly are you updating here?

Reply:
if I correctly understood how the method get_or_create() works. it is possible to update the second argument of the function, that is, in this case, a bonus